lib/remap-lang/src/lib.rs

```diff
@@ -281,6 +281,11 @@ mod tests {
                     "g": r#"{"1": Integer(1), "true": Boolean(true)}"#,
                 })),
             ),
+            ("true * 5 ? 5 * 5", Ok(()), Ok(value!(25))),
+            ("5 * 5 ? true * 5", Ok(()), Ok(value!(25))),
+            ("false * 5 ? true * 5 ? 5 * 5", Ok(()), Ok(value!(25))),
```

Comment:
Does this also allow for `()` grouping? That is, could I write `(false * 5) ? (true * 5 ? 5)`?

Reply:
I'll add a test, but it should, yes.